pandunia-lekse.csv

```diff
@@ -1326,6 +1350,7 @@ kamila||||perfect (optimize, refine, polish)|||||||||||||||||
 kamili|||ara:(kāmil), fas:(kâmel), urd:(kāmil), swa:kamili|perfect||perfecto||||||||पूर्ण|||||||perfekcyjny
 kamise|||spa:por:camisa, ara:(qamīṣ), hin:क़मीज़ (kāmīz), may:kemeja|shirt|chemise|camisa|camisa|рубашка|قَمِيص‎|衬衫|シャツ|셔츠||क़मीज़|শার্ট|kemeja|shati||ĉemizo|paita|koszula
 kamisi xake||||sleeve|manche|manga|manga|||袖子|袖||||||||||
+kamomile|||eng:camomile, por:camomila, fra:camomille, jpn:カモミール (kamomīru), may:kamomil|German camomile|Camomille sauvage|manzanilla de Castilla|camomila-vulgar|ромашка аптечная||母菊|カモミール (カミツレ)||||||||||
```

Comment:
Looks like a compound word. How about "kamile" from German "Kamille"?

Reply:
Wouldn't that conflict with "kamili"? The original Latin word was "chamaemelon", so perhaps "kamemile" would make the most sense.

Reply:
You're right, it conflicts with "kamili". However now I think that "kamili" was an unnecessary word. "kāmil" means "complete, full, whole, entire, total" in Arabic. I think that "puri" covers most of those meanings. So I will erase "kamili". In addition, I will introduce "purboni" (completely good) to cover one of the many meanings of "perfect".